SimG4CMS/HGCalTestBeam/test/HGCalTBGenSimDigiReco_cfg.py

```diff
@@ -110,7 +111,7 @@
 for path in process.paths:
         getattr(process,path)._seq = process.generator * getattr(process,path)._seq
 
-for label, prod in process.producers_().iteritems():
+for label, prod in process.producers_(six.iteritems()):
```

Comment:
I don't think this one is right... similar issue in the next two files below, and several others (search for `six.iteritems()`)

Reply:
indeed, that is wrong.  my script didn't handle it correctly. thanks for noticing